docs/ecosystem/integrations-community.md

```diff
@@ -39,6 +39,8 @@ Below are a list of community plugins and integrations created with Mermaid.
   - [Mermaid plugin for GitBook](https://github.com/wwformat/gitbook-plugin-mermaid-pdf)
 - [LiveBook](https://livebook.dev) ✅
 - [Atlassian Products](https://www.atlassian.com)
+  - [Mermaid Charts & Diagrams for Confluence](https://marketplace.atlassian.com/apps/1222572/)
+  - [Mermaid Charts & Diagrams for Jira](https://marketplace.atlassian.com/apps/1224537/)
```

Comment:
Nothing against it, but this is a bit out of the scope for the task. Let it be here, since you've already added it, but it's better to keep PR directly related to the issue, that would make tracking  the issue down much easier

Reply:
Oh - yeh - I agree. I'm not sure why it is here either? I haven't added it on purpose - has it come through on a merge? More than happy for them to be removed.

Reply:
It is probably originates from the merge commit, yep

Reply:
There was a commit which didn't update the docs. Most probably this was added by the pre-commit hook. 
You can ignore these. It will go away once you sync the latest develop.